assignment4/src/app.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import * as BlogController from './controllers/BlogController';
+import * as PermissionMiddleware from './PermissionMiddleware';
+
+const app = express();
+const PORT = 8080;
+
+app.use(bodyParser.json());
+
+app.get('/blogs', BlogController.getAllBlogs);
```

Comment:
we can make a single route directory and use that route here rather than all defining into app.ts.